Add horizontal support

oyvindsam opened the pull request:
Thanks for this package! I needed horizontal support, this is a quick implementation of that.

jesusrp98 commented:
Thanks for the feature implementation @oyvindsam ! And sorry it took so long for me to check out.
I'll take a look at the code now, would you be able to post some examples of how this feature works?

oyvindsam commented on example/lib/main.dart:
```diff
@@ -81,6 +81,47 @@ class HomePage extends StatelessWidget {
+                    style: Theme.of(context).textTheme.headline6,
+                  ),
+                  SizedBox(height: 8),
+                  Row(
```
What's changed here is basically just adding Row instead of Column, and setting vertical to false

oyvindsam commented on example/lib/main.dart:
```diff
@@ -81,6 +81,47 @@ class HomePage extends StatelessWidget {
+                      ),
+                      ExpandChild(
+                        vertical: false,
+                        child: Row(
```
Row

oyvindsam commented:
> Thanks for the feature implementation @oyvindsam ! And sorry it took so long for me to check out.
> I'll take a look at the code now, would you be able to post some examples of how this feature works?
> 
> Been trying to check its behaviour with no success :/

Sure thing, added an example!

jesusrp98 commented:
OK I see what you've done now. I've encountered a failure here in the example app.
<img width="424" alt="image" src="https://user-images.githubusercontent.com/8961959/106731477-cde7df80-660f-11eb-8d65-f0964dc78298.png">

Basically it overexands. We need to fix this, maybe wrapping the child inside an exanded widget?
Let me know if you need help here

oyvindsam commented:
Hm yes I did not account for that, I updated the example with how I implemented it in my other app. I am not sure how it would be done if expand_widget were to handle it itself.

jesusrp98 commented:
Hey @oyvindsam . I've updated the expansion API. Now it's using the `Axis` enum variable (`horizontal` or `vertical`) and it's vertical by default.

Check it out and I'll merge it to `master`!

Also, regarding the fix to the example, I think that could work. The `ExpandChild` widget shouldn't not include the `SingleChildScrollView` widget inside, it should be an external solution to it.

Massive thanks for supporting this package!!

oyvindsam commented:
Cool, looks good!
